webrtc.html

```diff
@@ -8852,6 +8854,34 @@ <h2 id="sec.stats-model">Statistics Model</h2>
       [[!WEBRTC-STATS]].</p>
     </section>
     <section>
+      <h3>The stats selection algorithm</h3>
+      <p>
+	The <dfn>stats selection algorithm</dfn> is as follows:
+	<ul>
+	  <li>If the selector is a MediaStreamTrack, return the following objects:
+	    <ul>
+	      <li>
+		The RTCMediaStreamStats corresponding to the selector.
+	      </li>
+	      <li>
+		If the selector track is inbound, the RTCInboundRTPStreamStats corresponding to the selector.
```

Comment:
That's derived from [RTCRTPStreamStats](https://w3c.github.io/webrtc-stats/#dom-rtcrtpstreamstats) which has the following foreign-key references: `associateStatsId`, `mediaTrackId` and `transportId` today.

I think we need an algorithm that (recursively?) includes all of these stats objects as well. Do others feel differently?